arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -184,17 +136,11 @@ def to_jobfile(self) -> str:
 
 @dataclass
 class StressNGParams:
-    """
-    The parameters in this schema will be passed through to the stressng
-    command unchanged
-    """
-
-    timeout: str = field(
-        metadata={
-            "name": "Runtime",
-            "description": "Time to run the benchmark test",
-        }
-    )
+    timeout: typing.Annotated[
+        str,
+        schema.name("Runtime"),
+        schema.description("Time to run the benchmark test"),
+    ]
```

Comment:
Actually, I think using "Runtime" as the schema name for "timeout" is bizarre in the first place. While certainly related concepts, most of the schema names are closely related to the parameter name. (Usually either the same token with `-` instead of `_`, like `hdd_ops`/`hdd_ops`, or an English phrase that's more descriptive, as in the params union below.)

Reply:
That's a good point.  I would recommend trashing both of them and going with something like "duration", or, perhaps, "interval" or "period".